pkg/ddl/000038_oidc_user.up.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+ALTER TABLE auth_users
+    ADD COLUMN IF NOT EXISTS external_id VARCHAR(255) UNIQUE;
```

Comment:
1. Any reason to use varchar and not text?
2. Missing drop column in the ...down.sql.

Reply:
1. Changed to TEXT
2. According to @arielshaqed, living the down migration empty.